src/index.js

```diff
@@ -272,14 +323,86 @@ export function FCCResetTests(suite) {
   suite.suites.forEach(FCCResetTests);
 }
 
-// Shortcut keys.
-// TODO: Need better inline docs on why we need to redefine the global
-// onkeydown and onkeyup.
+let count = -1;
+let parentTitle, mochaReport, mochaTimeStamp = null;
+
+/* In order to append test results to the shadow DOM, we need to use a custom
+ reporter. */
+function appendTestResult(test) {
+  /* Test Suite title */
+  var parentParentTitle = test.parent.parent.title;
+  var mainTitleNode =
+    shadow.querySelector('.fcc_test_message-box-header .title');
+  /* Append #mocha-report if it doesn't exist already */
+  if (!mochaReport) {
+    mainTitleNode.innerHTML = parentParentTitle;
+    mochaReport = document.createElement('ul');
+    mochaReport.setAttribute('id', 'mocha-report');
+    shadow.querySelector('.fcc_test_message-box-body #mocha')
+      .appendChild(mochaReport);
+  } else {
+    mochaReport = shadow.querySelector('#mocha-report');
+  }
+  /* If current test.parent.title is different from previous test, count up and
+    create a new section with testList */
+  if (!parentTitle || parentTitle !== test.parent.title) {
+    count++;
+    parentTitle = test.parent.title;
+    var testSection = document.createElement('li');
+    testSection.setAttribute('class', `fcc_section_${count} suite`);
+    var parentTitleNode = document.createElement('h1');
+    parentTitleNode.innerText = parentTitle;
+    testSection.appendChild(parentTitleNode);
+    /* Currently using <ul>, but if we switch to <ol> we can eliminate all of
```

Comment:
This is a good idea. I will create an issue for this. Can you just change this comment to start `/* TODO: Currently using...`